shared/control/parallel_dd.control

```diff
@@ -12,13 +12,8 @@ the files system.
 """
 
 # YOU NEED TO SPECIFY A FILESYSTEM
-path = glob.glob("/dev/[hsv]d?")
-path.sort()
-fs = path[-1]
-platform = Popen(["uname", "-m"], stdout=PIPE).communicate()[0]
-if "ppc" in platform:
-    if not Popen("lsscsi", stdout=PIPE).communicate()[0]:
-        fs = path[0]
+path = glob.glob("/dev/disk/by-id/*TARGET_DISK*")
+fs = ('/dev/' + Popen(["ls", path[0], "-ls"], stdout=PIPE).stdout.readline().split('/')[-1]).split('\n')[0]
```

Comment:
I think fs = ('/dev/' + Popen(["ls", path[0], "-ls"], stdout=PIPE).stdout.readline().split('/')[-1]).strip("\n") will be better.

Reply:
Good advice , I will update it , thank you very much.